### DISCUSSION TITLE
Ticket as pdf attached to email?

### DISCUSSION THREAD

**Discussion by holgerm** [pretix/pretix#3368] [Q&A]
Can I make Pretix send pdf tickets automatically with any of its emails, e.g. confirmation email? Would be great because explicit downloading can easily be forgotten and then check-in might be difficult at the event.

**Comment by pc-coholic** [comment]
pretix is attaching tickets by to emails if
- `Allow users to download tickets` is enabled in Settings -> Tickets
- The `PDF` output format is enabled
- The order position has a ticket template (or default template) assigned/ticket generation has not explicitly disabled for the item
- The `Download date` on Settings -> Tickets has been reached or is unset
- `Attach ticket files` in Settings -> E-Mail is enabled
- and most importantly: The total of all attachments is not too large

pretix will make an estimate, if the sum of all attachments is too big. If that's the case, the messages will be sent without the attached PDFs in order to make sure, that the customer is getting at least the link to their order rather than have the mail disappear by overzealous mail filters/servers.

If you do not receive attached tickets and have all the above things verified, the size is probably the issue. In this case, please make sure that your PDF contains vector graphics only. Also keep in mind, that all attachments are taken into consideration - so uploading giant pictures for passbook-files will cause the same issue.
